app/plugin_api.go

```diff
@@ -174,7 +174,15 @@ func (api *PluginAPI) CreateTeamMember(teamId, userId string) (*model.TeamMember
 }
 
 func (api *PluginAPI) CreateTeamMembers(teamId string, userIds []string, requestorId string) ([]*model.TeamMember, *model.AppError) {
-	return api.app.AddTeamMembers(teamId, userIds, requestorId)
+	if members, err := api.app.AddTeamMembers(teamId, userIds, requestorId, false); err != nil {
```

Comment:
Would you please change the code to have idiomatic error handling? See https://developers.mattermost.com/blog/idiomatic-error-handling/ for more context.